dotcom-rendering/src/web/components/Contributor.tsx

```diff
@@ -15,25 +16,59 @@ import TwitterIcon from '../../static/icons/twitter.svg';
 import { interactiveLegacyClasses } from '../layouts/lib/interactiveLegacyStyling';
 import { decidePalette } from '../lib/decidePalette';
 
-const twitterHandleStyles = (palette: Palette) => css`
-	${textSans.xxsmall()};
-	font-weight: bold;
+const defaultTwitterHandleColour = (palette: Palette) => css`
 	color: ${palette.text.twitterHandle};
 
-	padding-right: 10px;
-	display: inline-block;
+	svg {
+		fill: ${neutral[46]};
+	}
+
+	a {
+		color: ${palette.text.twitterHandle};
+	}
+`;
+
+const twitterHandleColour = (palette: Palette, format: ArticleFormat) => {
+	switch (format.design) {
```

Comment:
I think - if I'm reading this right - this switch needs to happen in `decidePalette`?